docs/modules/deployment/pages/upgrade/introduction.adoc

```diff
@@ -0,0 +1,53 @@
+
+[[upgrade]]
+= Upgrade {page-component-title}
+
+This section outlines the steps to follow to upgrade to a newer version of {page-component-title}.
+You may need to complete additional steps to upgrade in a more complex setup (running more than one OpenNMS instance, database migration requirements, age of current version, and so on).
+
+A basic upgrade with one instance takes about an hour.
+This time varies, and will be longer depending on how customized and complex your setup is.
+
+[[byb_upgrade]]
+== Before you begin
+
+There are a number of tasks you should complete before starting an upgrade:
+
+* Check the https://vault.opennms.com/docs/opennms/releases/index.html[release notes] for breaking changes in each release between your current version and the version you want to upgrade to.
```

Comment:
There is nothing in the vault starting with v28, and trying v28 on the docs site redirects to v29.  Would need to use github branches to get v28 (and v29 once 30 releases)

Reply:
Agreed. I'm hoping that we will have this issue addressed before 30 goes out, but I will add a note here about 28 in case we don't.